doc/en/monitoring/user-portal.rst

```diff
@@ -6,17 +6,17 @@ User portals
 Some clusters offer a portal to monitor your current jobs and even your
 recently completed jobs. Here they are:
 
-- **Béluga** : https://portail.beluga.calculquebec.ca
-- **Narval** : https://portail.narval.calculquebec.ca
-- **Niagara** : https://my.scinet.utoronto.ca
+- **Narval**: https://portail.narval.calculquebec.ca
+- **Rorqual**: https://metrix.rorqual.calculquebec.ca
+- **Trillium and Trillium GPU**: https://my.scinet.utoronto.ca
 
 In all cases, you must log in with your `CCDB <https://ccdb.alliancecan.ca>`__
 portal credentials.
 
 Narval job statistics
```

Comment:
Je dirais « Rorqual and Narval job statistics », considérant l’importance de Rorqual. Cela permettrait aussi de simplifier la phrase qui suit en enlevant les noms des grappes.